packages/rocketchat-theme/assets/stylesheets/rtl.less

```diff
@@ -191,22 +212,181 @@
 				border-radius: 5px 0 0 5px;
 				text-align: right;
 			}
-			>.users-typing {
+			> .users-typing {
 				float: right;
+			 }
+			> .formatting-tips {
+				float: left;
+				position: relative;
+				q {
+					padding: 0 3px 0 0;
+					border-right: 3px solid;
+					border-left: 0px none;
+					border-right-color: #CCC;
+				}
 			}
 		}
-		> .formatting-tips {
-			float: left;
+	}
+
+	.account-box {
+		.options {
+			direction: ltr;
+			> .wrapper {
+				direction: rtl;
+			}
 		}
-		.icon-paper-plane {
-			.left(10px);
-			transform: rotateY(180deg);
+	}
+
+	.flex-tab-bar {
+		left: 0;
+		right: unset;
+		border-right: 1px solid #eaeaea;
+		border-left: none;
+		z-index: 13;
+
+		.tab-button {
+			&.active {
+				margin-right: -1px;
+				margin-left: unset;
+				border-left: 3px solid #ff0000;
+				border-right: unset;
+			}
+		}
+	}
+
+	.flex-tab {
+		border-left: unset;
+		border-right: 1px solid;
+		right: unset;
+		left: 0;
+		.transform(translateX(-100%));
+		.control {
+			text-align: right;
+			padding: 12px 30px 12px;
+			> a,
+			> form {
+				float: right;
+			}
 		}
 	}
-	q {
-		.padding-right(10px);
+
+	.input-line {
+		&.search {
+			.octicon {
+				left: unset;
+				right: 10px;
+			}
+			input {
+				padding-left: unset;
+				padding-right: 20px;
+			}
+		}
+	}
+
+	.user-image {
+		.avatar {
+			&:after {
+				left: unset;
+				right: -12px;
+			}
+		}
+	}
+	.lines .user-image {
+		a {
+			> div {
+				float: right;
+			}
+		}
+		p {
+			float: right;
+			padding-right: 10px;
+			padding-left: unset;
+		}
 	}
-	q:before {
```

Comment:
@engelgabriel It was not correct and didn't apply to the blockquote, but I forgot to fix it with this patch.

I have it fixed with other issues I found later and will include them in another patch